backend/backend/settings.py

```diff
@@ -83,8 +86,8 @@
     },
 ]
 
-WSGI_APPLICATION = 'backend.wsgi.application'
-
+#WSGI_APPLICATION = 'backend.wsgi.application'
```

Comment:
let remove commented code, please! This can be found from the version control if we find we should use wsgi instead, which I doubt.